### PR TITLE
5838 Pension Pot Size Filter

### PR DESCRIPTION
**Pull request by lshepstone**
Support for indexing investment size IDs to support filtering for a pension pot size.

### CODE REVIEW

File: spec/serializers/firm_serializer_spec.rb [hunk 1]
```diff
@@ -32,9 +32,12 @@
       expect(subject[:wills_and_probate]).to be
     end
 
-    it 'exposes names of `other_advice_methods`' do
-      expect(subject[:other_advice_methods]).
-        to eql(firm.other_advice_method_ids)
+    it 'exposes `other_advice_method_ids`' do
```
Comment (esbel) [inline spec/serializers/firm_serializer_spec.rb]: thanks for that :)

**Comment by esbel** [comment]
:+1:

**Comment by benlovell** [comment]
:+1:

File: spec/serializers/firm_serializer_spec.rb (hunk 1 above)
Comment (lshepstone) [inline spec/serializers/firm_serializer_spec.rb]: np! ;)

**Pull request merged**